src/test/java/TestClass.java

```diff
@@ -0,0 +1,17 @@
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class TestClass {
+    @Test
+    public void generalTestCase() {
+        assertEquals(true, true);
```

Comment:
This was more to verify that tests were actually running, and the test below is to make sure the JNI linking actually works. I'll remove the general one, and move the JNI one to it's own package.